### ISSUE TITLE
CHIVES - DNS resolver

### ISSUE THREAD

**Issue by larsbrinkhoff** [PDP-10/its#429]
MC: CHIVES;  
https://www.hactrn.net/hacks/chives/

This is interesting, because it's substantial PDP-10 program written in C.  By @sraustein.

Depends on #419.

**Comment by larsbrinkhoff** [comment]
Looks like it was primarily a TOPS-20 program, but maybe ported to ITS?

https://www.ietf.org/rfc/rfc2308.txt

**Comment by sraustein** [comment]
It was intended that it also run on ITS, but we ran low on ITS machines by the time it was working.

**Comment by larsbrinkhoff** [comment]
Thanks @sraustein.  From the looks of it, it was supposed to be compiled with KCC, not the older C10.  Do you know if KCC ever made it to ITS?

**Comment by sraustein** [comment]
Yes, it was written assuming KCC, and no, as far as I can remember, KCC never made it to ITS.  Which was one of the main reasons why CHIVES never did either.
